### DISCUSSION TITLE
Does JSOn report displays Throughput for each request?

### DISCUSSION THREAD

**Discussion by priya-dsp** [artilleryio/artillery#2792] [Artillery]
Hi @hassy , In json report where can we find throughput [actual number of requests/minute the server handled] for each http requests?

**Comment by hassy** [comment]
hi @priya-dsp 👋 The JSON report will contain per URL metrics if you use the [`metrics-by-endpoint`](https://www.artillery.io/docs/reference/extensions/metrics-by-endpoint) plugin. One of those metrics is the number of HTTP response codes per URL, which provide the throughput information - both in aggregate over the duration of the entire test run, and in 10s intervals.
